readme.md

```diff
@@ -0,0 +1,48 @@
+[![Build Status](https://ci.cloudnetservice.eu/buildStatus/icon?job=CloudNetService/v2-chatsystem/master)](https://ci.cloudnetservice.eu/job/CloudNetService/job/v2-chatsystem/master)
+[![star this repo](http://githubbadges.com/star.svg?user=CloudNetService&repo=v2-chatsystem)](https://github.com/CloudNetService/v2-chatsystem)
+[![fork this repo](http://githubbadges.com/fork.svg?user=CloudNetService&repo=v2-chatsystem)](https://github.com/CloudNetService/v2-chatsystem/fork)
+[![GitHub license](https://img.shields.io/github/license/CloudNetService/v2-chatsystem.svg)](https://github.com/CloudNetService/v2-chatsystem/blob/master/LICENSE)
+
+[![DepShield Badge](https://depshield.sonatype.org/badges/CloudNetService/v2-chatsystem/depshield.svg)](https://depshield.github.io)
+[![GitHub issues](https://img.shields.io/github/issues/CloudNetService/v2-chatsystem.svg)](https://github.com/CloudNetService/v2-chatsystem/issues)
+[![GitHub contributors](https://img.shields.io/github/contributors/CloudNetService/v2-chatsystem.svg)](https://github.com/CloudNetService/v2-chatsystem/graphs/contributors)
+[![Github All Releases](https://img.shields.io/github/downloads/CloudNetService/v2-chatsystem/total.svg)](https://github.com/CloudNetService/v2-chatsystem/releases)
+[![GitHub release](https://img.shields.io/github/release/CloudNetService/v2-chatsystem.svg)](https://github.com/CloudNetService/2-chatsystem/releases)
+
+
+# Chat System | The Cloud Network Environment Technology 2
+![Image of CloudNet](https://cdn.discordapp.com/attachments/325383142464552972/354670548292206594/CloudNet.png)
+
+This is the SimpleNametag System for CloudNet 2.2.0
+ 
+
+ ### Support
+ 
+  * Spigot-Support » 1.8 - 1.15.X
```

Comment:
Given the api-version in the plugin.yml, this seems incorrect.

Reply:
Is correct based on https://www.spigotmc.org/wiki/plugin-yml/ wiki page